logger.js

```diff
@@ -1,22 +1,23 @@
 'use strict';
 // Log class
 
-var util      = require('util');
-var tty       = require('tty');
+const util      = require('util');
+const tty       = require('tty');
 
-var constants = require('haraka-constants');
+const constants = require('haraka-constants');
 
-var config    = require('./config');
-var plugins;
-var connection;
-var outbound;
+const config    = require('./config');
+/* eslint-disable prefer-const */
```

Comment:
Why the disable?

Reply:
because when I added that, I hadn't yet found this option:
````
"prefer-const": ["error", {"ignoreReadBeforeAssign": true}],
````
I'll remove it (it's not longer needed).